src/main/java/com/squareup/javapoet/TypeName.java

```diff
@@ -168,6 +169,10 @@ public TypeName unbox() {
     throw new UnsupportedOperationException("cannot unbox " + this);
   }
 
+  public final TypeMirror toTypeMirror(Elements elements) {
```

Comment:
I think this goes in Poetry too . . . mostly because the documentation necessary to explain all of the failure cases is way larger than the method body itself.